arkouda/pdarraysetops.py

```diff
@@ -90,9 +95,9 @@ def _in1d_single(
         # this causes an issue with mypy for unknown reasons.
         if pda1.size == 0:
             return zeros(0, dtype=akbool)
-    if isinstance(pda2, pdarray) or isinstance(pda2, Strings) or isinstance(pda2, Categorical_):
-        if pda2.size == 0:
-            return zeros(pda1.size, dtype=akbool)
+        if isinstance(pda2, pdarray) or isinstance(pda2, Strings) or isinstance(pda2, Categorical_):
+            if pda2.size == 0:
+                return zeros(pda1.size, dtype=akbool)
```

Comment:
was this meant to be indented?

Reply:
Yes, if you look at the logic it only makes sense if both `pda1` and `pda2` are `(pdarray, Strings, Categorical)` (not a Sequence), which mypy will complain about.  Indenting was the easy way of achieving this.

Reply:
makes sense! I like the change!